virttest/utils_logfile.py

```diff
@@ -156,7 +156,7 @@ def close_log_file(filename):
                            " _open_log_files")
     try:
         for log_file, log_fd in _open_log_files.items():
-            if os.path.basename(log_file) == os.path.basename(filename):
+            if not filename or os.path.basename(log_file) == os.path.basename(filename):
```

Comment:
Suggest as the following then we can get rid of adding a new function.

```python
def close_log_file(filename='*'):
    ...
        for log_file, log_fd in _open_log_files.items():
            if (filename == '*' or
                os.path.basename(log_file) == os.path.basename(filename)):
    ...
```

Reply:
Do you mean `close_log_files` here as the proper name? A wildcard indicates we can close more than a single file. Also which functions you are referring to as redundant? The additional functions are all simple wrappers that either make the use of an obligatory `filename` param or don't use it all, I think all of them so far have clear name and intent.

Reply:
Sorry for the confusion. My intention was, we could introduce the wildcard `*` to the existing function `close_log_file` for extending its ability on closing all the log handlers managed by the `_open_log_files` list, and that could avoid adding a new function (i.e. the `close_log_files`).

The reason why I preferred as so is that, I feel like that the user may always using the `close_log_files` instead of another with having the both.

Reply:
Alright, it is clear now and I added the changes.